next.config.js

```diff
@@ -8,8 +8,13 @@ const myAppDirectives = {
   "connect-src": ["'self'", "rt6o382n.api.sanity.io", "rt6o382n.apicdn.sanity.io"],
 };
 
+const isProd = process.env.NODE_ENV === "production";
+
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  // Use the CDN in production and localhost for development.
+  assetPrefix: isProd ? "https://cdn.nav.no/teamdagpenger/dp-produktside-frontend/<env>" : undefined,
```

Comment:
```suggestion
  assetPrefix: isProd ? "https://cdn.nav.no/teamdagpenger/dp-produktside-frontend" : undefined,
```

Du trenger nok ikke *env* her

Reply:
ah glemte å endre den